### PR TITLE
Fix ContentRadioList import

### PR DESCRIPTION
**Pull request by JoseOrtiz**
## Description
Add import to library.js 

**Extra**: Installed `addon-controls` and added a prototype for controls on ContentRadioList

## Checks
- [ ] Requires documentation update
- [ ] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [ ] **MINOR** adds functionality in a backwards-compatible manner.
- [x] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: src/lib/RenderString.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      components: {
+        ...AllIcons,
+      },
+      template: `<div>${this.string}</div>`,
+    };
+    return createElement(render);
+  },
+};
+
+export default RenderString;
```
Comment (TanMar23) [inline src/lib/RenderString.js:21]: This file should be named after camelCase standard

File: src/components/ContentRadioList/ContentRadioList.stories.js [hunk 2]
```diff
@@ -1,149 +1,151 @@
...
+      this.selectedValue = this.value;
+    },
+  },
   template: `
-    <div style="margin: 10px 50px 10px 50px;">
-      <h2><strong>ContentRadioList:</strong>&nbsp;A control that allows a user to select an option by showing all available options as a list of radio buttons.</h2>
+    <div>
+      <ContentRadioList v-model="selectedValue"
+                        :id="id"
+                        :dataTestId="dataTestId"
```
Comment (TanMar23) [inline src/components/ContentRadioList/ContentRadioList.stories.js:91]: This should be :data-test-id="dataTestId"

**Comment by TanMar23** [comment]
In general it looks good to me, but I just have a question. I don't understand how are we going to select between checked or unchecked since I see that in the code (within Docs) is pretty much the same, so I was wondering if there will be maybe a prop or something like that to distinguish between them
image

File: src/lib/RenderString.js (hunk 1 above)
Comment (JoseOrtiz) [reply src/lib/RenderString.js]: changed

File: src/components/ContentRadioList/ContentRadioList.stories.js (hunk 2 above)
Comment (JoseOrtiz) [reply src/components/ContentRadioList/ContentRadioList.stories.js]: changed

**Comment by JoseOrtiz** [comment]
> In general it looks good to me, but I just have a question. I don't understand how are we going to select between checked or unchecked since I see that in the code (within Docs) is pretty much the same, so I was wondering if there will be maybe a prop or something like that to distinguish between them
> image

I hardcoded the source code, as dynamic source code isn't [supported for Vue](https://storybook.js.org/docs/react/api/frameworks-feature-support)

**Review by TanMar23 — APPROVED** [review]

**Pull request merged**